Assets/Content/Furniture/Machines/Science/CellCharger.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using SS3D.Content.Furniture.Storage;
+using SS3D.Content.Systems.Interactions;
+using SS3D.Engine.Interactions;
+using SS3D.Engine.Interactions.Extensions;
+using SS3D.Engine.Inventory;
+using UnityEngine;
+using Mirror;
+
+[RequireComponent(typeof(Container))]
+public class CellCharger : NetworkBehaviour
+{
+    private StorageContainer storageContainer;
+    private Container container;
+    private Item powerCell;
+    public MeshRenderer renderer;
+
+    private void Start()
+    {
+        container = GetComponent<Container>();
+        StartCoroutine("StartCharge");
```

Comment:
This should be wrapped in an isServer check.
```suggestion
        if (isServer)
        {
            StartCoroutine(StartCharge());
        }
```